src/ethereum/frontier/state.py

```diff
@@ -35,6 +35,53 @@ class State:
         default_factory=lambda: Trie(secured=True, default=None)
     )
     _storage_tries: Dict[Address, Trie[U256]] = field(default_factory=dict)
+    _snapshots: List[
+        Tuple[Trie[Optional[Account]], Dict[Bytes, Trie[U256]]]
+    ] = field(default_factory=list)
+
+
+def start_transaction(state: State) -> None:
```

Comment:
```suggestion
def begin_transaction(state: State) -> None:
```

the other functions match SQL, so why not this one!